tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +15,128 @@
           preparsing_validators=[
               validators.hasLength(71),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.notMatches(1),
+                  result_field='SSN', result_function=validators.isNumber()
```

Comment:
the notes on this one leave me questioning the `result_function`

> if TANF recipient, then cannot have all 0s,1s,2s,3s,4s,5s,6s,7s,8s,or 9s for SSN.

#2518 handles it more like i'd expect
https://github.com/raft-tech/TANF-app/blob/4b70e6aa36b0cc605579d38e6eabf679fe3811b7/tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py#L18